pulp_2_tests/tests/platform/api_v2/test_consumer.py

```diff
@@ -67,3 +70,70 @@ def test_all(self):
             self.assertEqual(result['consumer_id'], consumer['consumer']['id'])
             self.assertEqual(result['distributor_id'], body['distributor_id'])
             self.assertEqual(result['repo_id'], body['repo_id'])
+
+
+class RegisterAndUpdateConsumerTestCase(unittest.TestCase):
+    """Show that one can `register and update a consumer`_.
+
+        The call to `consumer register api` should return a x.509 certificate,
```

Comment:
This isn't how to create a link:

    My `link`.

This is:

    My `link`_.